base/uk.ac.stfc.isis.ibex.ui.help/resources/iconlicences.txt

```diff
@@ -0,0 +1,234 @@
+
+
+Perspective Switcher
+
+->ui.perspectiveswitcher/icons
+reset icon designed by Freepik at www.flaticon.com 
+
+
+
+Scripting
+
+->ui.scripting/icons
+alarm icon designed by Freepik from www.flaticon.com
```

Comment:
Maybe it's a good idea to explicitly give the full file path for each icon. Either:

```
/uk.ac.stfc.isis.ibex.ui.scripting/icons/script_32x24.png alarm icon 
designed by Freepik from www.flaticon.com

/uk.ac.stfc.isis.ibex.ui.scripting/icons/example_32x24.png alarm icon 
designed by Freepik from www.flaticon.com
```

Or, to save some space and be a bit less verbose

```
Beam Status
Located in uk.ac.stfc.isis.ibex.ui.beamstatus/icons/

status_24x24 designed by Freepik from www.flaticon.com
status_32x24 designed by Freepik from www.flaticon.com
```

Similarly to how you are doing it, just a little more obvious.

Reply:
As per our conversation: I have updated the icon path to have a full directory path and placed these inline with the sub-headings.